From 7285746f8757de77eea45bb23b5cef18d3b98cc7 Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Fri, 1 Oct 2021 08:52:39 +0200
Subject: [PATCH 2/2] Disable Python 2.7

---
 CMakeLists.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index df03743..a7215ad 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,13 +21,6 @@ include(CMakeGeneratePkgConfig REQUIRED)
 
 find_package(PythonEasy REQUIRED)
 
-if (PYTHON_EXECUTABLE)
-    if(RHEL STREQUAL "7")
-      find_package(Boost COMPONENTS python REQUIRED)
-    endif()
-endif(PYTHON_EXECUTABLE)
-
-
 if (PYTHON3_CURRENT_VERSION)
     string(REPLACE "." "" Boost_PYTHON_3_SUFFIX "${PYTHON3_CURRENT_VERSION}")
 
-- 
2.33.0

