From f7151eb8700888e67bc7fcd753219dbfe7f3e5b8 Mon Sep 17 00:00:00 2001
From: Mihai Patrascoiu <mihai.patrascoiu@cern.ch>
Date: Tue, 26 Jul 2022 15:09:54 +0300
Subject: [PATCH] DMC-1330: Support for Python 3.11 build

---
 cmake/modules/FindPythonEasy.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmake/modules/FindPythonEasy.cmake b/cmake/modules/FindPythonEasy.cmake
index d85a81f..b64877c 100644
--- a/cmake/modules/FindPythonEasy.cmake
+++ b/cmake/modules/FindPythonEasy.cmake
@@ -15,7 +15,7 @@
 LIST(APPEND L_PYTHON_VERSIONS
     "1.5" "1.6"
     "2.0" "2.1" "2.2" "2.4" "2.5" "2.6" "2.7" "2.8"
-    "3" "3.0" "3.1" "3.2" "3.3" "3.4" "3.5" "3.6" "3.7" "3.8" "3.9" "3.10"
+    "3" "3.0" "3.1" "3.2" "3.3" "3.4" "3.5" "3.6" "3.7" "3.8" "3.9" "3.10" "3.11"
 )
 
 INCLUDE(FindPackageHandleStandardArgs)
@@ -28,6 +28,7 @@ INCLUDE(FindPackageHandleStandardArgs)
   HINTS
   ${ALT_PYTHON_LOCATION}/bin
   PATHS
+  [HKEY_LOCAL_MACHINE\\SOFTWARE\\Python\\PythonCore\\3.11\\InstallPath]
   [HKEY_LOCAL_MACHINE\\SOFTWARE\\Python\\PythonCore\\3.10\\InstallPath]
   [HKEY_LOCAL_MACHINE\\SOFTWARE\\Python\\PythonCore\\3.9\\InstallPath]
   [HKEY_LOCAL_MACHINE\\SOFTWARE\\Python\\PythonCore\\3.8\\InstallPath]
-- 
GitLab

